### DISCUSSION TITLE
I can view MQTT  but can't write, please help.

### DISCUSSION THREAD

**Discussion by agapitoangel** [john30/ebusd#991] [Q&A]
Good afternoon everyone, excuse my ignorance. I'm quite proficient with esphome, and I thought this would be something similar, but to my surprise, it's much more complex.

I've already installed eBUSd on Home Assistant, and the magic happened automatically. New sensors for my Vaillant appeared on MQTT. However, I'm unsure how to input or send information via MQTT.
![image](https://github.com/john30/ebusd/assets/96874244/b8a475df-0967-4799-ad44-3526c752176f)
![image](https://github.com/john30/ebusd/assets/96874244/ac67722c-873d-4a54-94db-56dee5071578)

All I need is to be able to control the heating and adjust the temperature of my Vaillant aerotherm. I've read multiple times about modifying files in etc.../templates.csv, but I can't seem to access those files. Do I need to access them through SSH?

I'm using Home Assistant OS. Thank you very much, and I apologize for any inconvenience.

I'm not sure if someone could create a video on YouTube; it would be a great way to get more people involved in this project.

Thank you in advance.

**Comment by CommanderROR9** [comment]
I can't really help, but can confirm that the file we are supposed to edit isn't in etc/ebusd folder. That folder doesn't exist

**Comment by CommanderROR9** [comment]
So...I think I managed this...I downloaded the MQTT-Hassio.cfg and copied it to /config/ebus/ and then configured the Addon to access it there. In that file I then made the necessary changes. Haven't tested anything yet, but that's my current idea

**Reply by Koky05** [reply]
At least you need to change `filter-direction = r|u|^w` to allow read also writable values in MQTT-Hassio.cfg.
